### PR TITLE
[Merged by Bors] - feat (Order/BoundedOrder): add bot_lt_of_gt

### PR DESCRIPTION
**Pull request by YnirPaz**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by vihdzp** [comment]
I've got two questions
1) Couldn't this be placed in the file defining `OrderBot`? It shouldn't depend on anything else.
2) Perhaps we should name this something like `bot_lt_of_lt` or `LT.bot_lt` instead? Even on a type with a `0` instance there's no guarantee `⊥ = 0`.

**Review by YaelDillies — COMMENTED** [review]
This already exists under the name `LT.lt.bot_lt`

**Comment by YnirPaz** [comment]
> This already exists under the name `LT.lt.bot_lt`

Sorry, I'm not seeing this?

**Comment by YaelDillies** [comment]
Oh I guess we only have `Ne.bot_lt`. Feel free to add yours as well under the name I gave above

File: Mathlib/Order/BoundedOrder.lean [hunk 1]
```diff
@@ -253,6 +253,11 @@ theorem ne_bot_of_gt (h : a < b) : b ≠ ⊥ :=
 
 alias LT.lt.ne_bot := ne_bot_of_gt
 
+theorem bot_lt_of_gt (h : a < b) : ⊥ < b :=
```
Comment (YaelDillies) [inline Mathlib/Order/BoundedOrder.lean:256]:
```suggestion
theorem bot_lt_of_lt (h : a < b) : ⊥ < b :=
```

Comment (YaelDillies) [inline Mathlib/Order/BoundedOrder.lean:256]: Can we have the dual lemma about top?

**Review by YaelDillies — APPROVED** [review]
maintainer merge

**Comment by fpvandoorn** [comment]
bors merge

**Pull request closed without merging**